rules/order.js

```diff
@@ -0,0 +1,101 @@
+module.exports = {
+  'order/properties-order': [
+    {
+      order: 'strict',
+      properties: [
+        'content',
+      ],
+    },
+    {
+      order: 'strict',
+      properties: [
+        'position',
+        'z-index',
+        'top',
+        'bottom',
+        'left',
+        'right',
+      ],
+    },
+    {
+      order: 'strict',
+      properties: [
+        'transform',
+        'transform-origin',
+      ],
+    },
+    {
+      order: 'strict',
+      properties: [
+        'box-sizing',
+        'display',
+        'overflow',
+        'flex',
+        'flex-grow',
+        'flex-shrink',
+        'flex-basis',
+        'order',
+        'align-self',
+        'flex-direction',
+        'flex-wrap',
+        'justify-content',
+        'align-items',
+        'align-content',
+        'height',
+        'width',
+        'margin',
+        'margin-top',
```

Comment:
Ah yes, this makes sense now. 

IIRC, in a casual conversation, a few months back in our pod @lemonmade brought up this same argument. I believe he agreed that what we do is weird too.  I forget what the original reasoning was, maybe he can shed some light on this.